src/NativeScript/BuiltinNames.cpp

```diff
@@ -0,0 +1,55 @@
+/*
```

Comment:
This file is not needed. It's accidentally been committed after trying various module loader fixes.